static-website/static-website.yaml

```diff
@@ -207,7 +213,7 @@ Resources:
         CustomErrorResponses: !If
         - HasDefaultErrorPagePath
         - - ErrorCode: 403 # not found
```

Comment:
I suggest to improve the comment here: `# 403 from S3 indicates that the file does not exists`